src/mds/OpenFileTable.h

```diff
@@ -69,8 +69,8 @@ class OpenFileTable
 
   version_t omap_version = 0;
 
-  static const unsigned MAX_ITEMS_PER_OBJ = 1024 * 1024;
-  static const unsigned MAX_OBJECTS = 1024; // billion items at most
+  unsigned MAX_ITEMS_PER_OBJ = g_conf()->osd_deep_scrub_large_omap_object_key_threshold;
```

Comment:
New code shouldn't use `g_conf()->confname`. We should `g_conf().get_val<uint64_t>("osd_deep_scrub_large_omap_object_key_threshold")` instead.

Reply:
thanks, Patrick. will change it.